lib/handlers/get.js

```diff
@@ -76,7 +77,13 @@ function handler (req, res, next) {
     // Till here it must exist
     if (!includeBody) {
       debug('HEAD only')
-      contentType = mime.contentType(_path.extname(path))
+      const rootUrl = utils.getBaseUri(req)
+      const mapper = new LegacyResourceMapper({
+        rootUrl,
+        rootPath: ldp.root,
+        includeHost: ldp.multiuser
+      })
+      contentType = mapper._getContentTypeByExtension(path)
```

Comment:
This is a private method though (underscore); you probably want `mapper.mapFileToUrl({ path }).contentType`.

Reply:
Ok, I'll do that ^_^